checkita-core/src/main/scala/ru/raiffeisen/checkita/storage/Models.scala

```diff
@@ -132,6 +152,7 @@ object Models {
                                          numMetrics: Int,
                                          numChecks: Int,
                                          numLoadChecks: Int,
+                                         numJobConfig: Int,
```

Comment:
I don't think we would ever need this metric. We always have sole instance of job configuration.